base_report_to_printer/views/res_users_view.xml

```diff
@@ -6,12 +6,12 @@
     <field name="model">res.users</field>
     <field name="inherit_id" ref="base.view_users_form" />
     <field name="arch" type="xml">
-      <page string="Preferences">
-        <group name="printing">
+      <xpath expr="//page[group/group[@name='preferences']]" position="inside">
```

Comment:
It would take the first, although you can put also indexes if needed, but it's not the usual case.